Lib/test/test_urllib.py

```diff
@@ -187,12 +187,15 @@ def test_close(self):
 
     def test_info(self):
         self.assertIsInstance(self.returned_obj.info(), email.message.Message)
+        self.assertIsInstance(self.returned_obj.headers, email.message.Message)
 
     def test_geturl(self):
         self.assertEqual(self.returned_obj.geturl(), self.pathname)
+        self.assertEqual(self.returned_obj.url, self.pathname)
 
     def test_getcode(self):
         self.assertIsNone(self.returned_obj.getcode())
+        self.assertIsNone(self.returned_obj.status)
```

Comment:
I think it could be cleaner to separate these tests (recommended attributes first, then the older ones)

Reply:
Ok.

In other tests that use the `info()`, `geturl()` functions, etc., do you suggest I update them to use the recommended attributes as well too?

Reply:
How many changes is that?

If the functions are doc-deprecated but will never actually raise warnings, maybe it’s not worth changing tens and tens of lines.